frontend/app/src/pages/main/tenant-settings/components/members-columns.tsx

```diff
@@ -39,5 +46,40 @@ export const columns = (): ColumnDef<TenantMember>[] => {
         <div>{relativeDate(row.original.metadata.createdAt)}</div>
       ),
     },
+    {
+      id: 'actions',
+      cell: ({ row }) => (
+        <MemberActions
+          row={row}
+          onChangePasswordClick={onChangePasswordClick}
+        />
+      ),
+    },
   ];
 };
+
+function MemberActions({
+  row,
+  onChangePasswordClick,
+}: {
+  row: Row<TenantMember>;
+  onChangePasswordClick: (row: TenantMember) => void;
+}) {
+  const { user } = useOutletContext<UserContextType>();
+
+  if (user.metadata.id === row.original.metadata.id) {
```

Comment:
@abelanger5 we should likely also guard here if its SSO, but currently this is not exposed on the user object. Any toughts?